lib/console.h

```diff
@@ -33,9 +33,15 @@
 #define CONSOLE_CYAN    "\033[36m"
 #define CONSOLE_WHITE   "\033[37m"
 
+#define CONSOLE_MIRRORS 3
+
 
 /* Sets console device */
-extern void lib_consoleSet(unsigned major, unsigned minor);
+extern void lib_consoleSet(unsigned int major, unsigned int minor);
+
+
+/* Sets console mirror devices */
+extern void lib_consoleSetMirrors(int major[CONSOLE_MIRRORS], int minor[CONSOLE_MIRRORS]);
```

Comment:
I'd prefer passing the number of mirrors explicitly (`void lib_consoleSetMirrors(unsigned int *major, unsigned int *minor, unsigned int cnt)`) to avoid type mismatches between `lib_consoleSet` and  `lib_consoleSetMirrors`.